deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/qa_provisioning.py

```diff
@@ -122,16 +136,27 @@ def __process_config_data(self, host_provision_info):
                 installation_files_parameters['wazuh_branch'] = wazuh_branch
                 installation_instance = WazuhSources(**installation_files_parameters)
             if install_type == 'package':
-                    if s3_package_url is None:
+
+                    if s3_package_url is None and local_package_path is None:
+                        installation_files_parameters['version'] = version
+                        installation_files_parameters['system'] = system
+                        installation_files_parameters['revision'] = revision
+                        installation_files_parameters['repository'] = repository
+                        installation_files_parameters['architecture'] = architecture
+                        installation_instance = WazuhS3Package(**installation_files_parameters)
+                        remote_files_path = installation_instance.download_installation_files(self.inventory_file_path,
+                                                                                              hosts=current_host)
+                    elif s3_package_url is None and version is None:
```

Comment:
??? Here you have to check if a local package has been specified

Reply:
This requested change can be reviewed in the commit : [5c3a34d2757275d66bb9951667083965a0452fab](https://github.com/wazuh/wazuh-qa/commit/5c3a34d2757275d66bb9951667083965a0452fab)